i-dot-ai/redbox
# Create automated sign-up form

## Context

To automate the sign-up process, this is creating the sign-up form in Django. It also incorporates the Common Task Survey.

https://technologyprogramme.atlassian.net/browse/REDBOX-702


## Changes proposed in this pull request

* Combine the sign-up form and common task survey into one, making this work with the Django User model
* Automatically create a user once the form is submitted
* Link to the form from the homepage


## How to test

* Add `ALLOW_SIGN_UPS=True` to your Django ENVs
* Click the sign-up link from the homepage and follow the sign-up process
* Check you can download the CSV with all the data from the Django Admin console


## Things to check

- [ ] I have added any new ENV vars in all deployed environments
- [ ] I have tested any code added or changed
- [x] I have run integration tests
